### DISCUSSION TITLE
Showing JSON file snippet that causes an error.

### DISCUSSION THREAD

**Discussion by ethanLhciactive** [CMSgov/price-transparency-guide-validator#117] [Q&A]
Hi, if input JSON is invalid, is there a way to show the snippet in the JSON file that causes the error?

**Comment by shaselton-usds** [comment]
@ethanLhciactive 

Sorry for the delayed response, but I think you can already do this by evaluating the error code that is passed back in conjunction with the `instanceRef` and `schemaRef` that are passed back. Let me know if you have further trouble in figuring what causes errors.
